types/mixpanel-browser/index.d.ts

```diff
@@ -3,6 +3,7 @@
 // Definitions by: Carlos López <https://github.com/karlos1337>
```

Comment:
Any idea why these files weren't already prettied?

Reply:
No idea, I joined this recently as you are doing here, and they were already like this.